packages/apollo/src/commands/service/check.ts

```diff
@@ -131,15 +131,22 @@ export function formatHumanReadable({
   if (changes.length === 0) {
     result = "\nNo changes present between schemas";
   } else {
-    const breakingChanges = changes.filter(
+    // Create a sorted list of the changes. We'll then filter values from the sorted list, resulting in sorted
+    // filtered lists.
+    const sortedChanges = sortBy<typeof changes[0]>(changes, [
```

Comment:
Does `sortBy` infer the type of `changes`?

I thought it would 😞

Reply:
Unfortunately no; it appears as though the array syntax causes confusion.